<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdiumOptionsWidgetClass</class>
 <widget class="QWidget" name="AdiumOptionsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>162</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="lblParameters">
     <property name="text">
      <string notr="true">Parameters</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblVariant">
     <property name="text">
      <string>Variant:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="buddy">
      <cstring>cmbVariant</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="cmbVariant">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblFont">
     <property name="text">
      <string>Font:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lneFont">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="tlbFontChange">
     <property name="text">
      <string>Change...</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="tlbFontReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="lblBackground">
     <property name="text">
      <string notr="true">Background</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblColor">
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="buddy">
      <cstring>cmbColor</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QComboBox" name="cmbColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblImage">
     <property name="text">
      <string>Image:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lneImage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>not set</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QToolButton" name="tlbImageChange">
     <property name="text">
      <string>Change...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QToolButton" name="tlbImageReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblImageLayout">
     <property name="text">
      <string>Layout:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="buddy">
      <cstring>cmbImageLayout</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QComboBox" name="cmbImageLayout">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
